src/themes/teams/components/List/listItemStyles.ts

```diff
@@ -37,12 +50,10 @@ const listItemStyles: IComponentPartStylesInput<IListItemProps, any> = {
     lineHeight: variables.headerLineHeight,
   }),
   headerMedia: ({ variables }): ICSSInJSStyle => ({
-    color: variables.headerMediaColor,
```

Comment:
@levithomason , please let me know if this is still needed? I can add it in the variable and use the override from the root styles. But anyways, I am not sure if having headerMediaColor in the variables is needed here since it should anyways get inverted when hovering for selection...